client/src/pages/Profile.js

```diff
@@ -1,6 +1,189 @@
-import React from 'react';
+import React, { useState, useContext, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { NextStepButton } from '../components/Buttons';
+import { useStyles } from '../themes/theme';
+import { store } from '../context/store';
+import { USER_LOADED } from '../context/types';
+import { Rating } from '@material-ui/lab';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
```

Comment:
We can combine the material ui items into a single import statement to clean this up a bit:
`impot { Select, MenuItem, Avatar, Button} from '@material-ui/core`